frontend/src/components/facilities/FacilityInfo.vue

```diff
@@ -170,6 +170,78 @@
             </v-row>
           </v-col>
         </v-row>
+
+        <v-divider />
+
+        <v-expansion-panels v-if="facility?.additionalAddresses.length > 0">
```

Comment:
since this expansion panel is for physical address, I think we should move it to right under physical address. Right now it looks like we have multiple mailing addresses. 
![image](https://github.com/bcgov/ecc-ofm/assets/119005543/2007bd43-f751-48fa-99b6-258581466f15)

Reply:
yup and done!